<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordField</class>
 <widget class="QWidget" name="PasswordField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>44</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_application">
     <property name="text">
      <string>application</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_login">
     <property name="text">
      <string>login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_password">
     <property name="text">
      <string>password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_delete">
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
